src/sample_package/launch/my_node.launch

```diff
@@ -1,5 +0,0 @@
-<launch>
```

Comment:
We should probably leave the `sample_package` around. You can get it back if you run `git checkout master -- src/sample_package` from the top level of your clone of this repo. (It will pull the `sample_package` folder from `master` back onto your branch.